lib/geordi/util.rb

```diff
@@ -182,6 +186,16 @@ def testing?
         !!ENV['GEORDI_TESTING']
       end
 
+      def irb_version
+        version_string = if testing?
+                           ENV['GEORDI_TESTING_IRB_VERSION']
+                         else
+                           `irb --version`.match(/\d\.\d/)
```

Comment:
When you're only matching two digits, how can you distinguish "1.2" and "1.2.6"? This really is "good luck matching". Also, this does not return a string, but a MatchData object.

Better:
```suggestion
                           `irb --version`[/irb (\d\.\d\.\d)/, 1]
```
This is a shorthand for "match this regex against this string, and return what was matched by parentheses group no 1.

Please also reduce the indentation to two spaces.